src/lunabot_perception/lunabot_perception/crater_detection.py

```
@@ -0,0 +1,260 @@
+        )
+
+    def _cloud_callback(self, msg: PointCloud2):
+        """Transform incoming cloud to odom frame and accumulate elevation."""
+        try:
+            tf = self.tf_buffer.lookup_transform(
+                self.target_frame,
+                msg.header.frame_id,
+                Time.from_msg(msg.header.stamp),
+                timeout=Duration(seconds=0.2),
+            )
+        except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)